### DISCUSSION TITLE
Multiple instances or parallel libreoffice convert

### DISCUSSION THREAD

**Discussion by bobitza** [gotenberg/gotenberg#626] [Q&A] [answered]
I observe if you convert for example 5 files and start in same time, each one wait for another one, if first is a big file, others have to wait alot and if exceed the timeout others are not working anymore

How to make gotenberg work in parallel ?

**Comment by gulien** [comment, marked as answer]
Hello @bobitza,

By default, there is a single long running instance of LibreOffice, which allows for faster unitary conversions.

To workaround your bottleneck, you may either:

* Launch many instances of Gotenberg
* Use the `--uno-listener-restart-threshold=0` (i.e., a dedicated LibreOffice listener per conversion)
